packages/fast-jss-utilities/README.md

```diff
@@ -28,6 +28,43 @@ const styles = {
 };
 ```
 
+### applyFocusVisible
+`applyFocusVisible` accepts style(s) and an optional selector string. `applyFocusVisible` automatically removes user-agent focus outlines by setting `outline: "none"`.
+
+This function implements the utility for all focus states, if you want to enable focus-visible support you should implement the focus-visible polyfill. Styles that are given will be returned with a focus selector that is based on whether focus-visible is supported natively by the browser, if focus-visible is not supported we apply selectors that mimic polyfill's behavior. If a selector has been passed, this will be added to the focus selector string for the styles. If you are padding a selector be sure to include a space before the selector so it appends properly.

```

Comment:
@eljefe223 can we make "focus-visible polyfill" below a link to the recommended polyfill? That way I don't have to go search for it.

> if you want to enable focus-visible support you should implement the focus-visible polyfill